base/server/src/main/java/org/dogtagpki/server/cli/SubsystemDBAccessGrantCLI.java

```diff
@@ -69,19 +62,8 @@ public void execute(CommandLine cmd) throws Exception {
 
         String dn = cmdArgs[0];
 
-        String catalinaBase = System.getProperty("catalina.base");
-
-        TomcatJSS tomcatjss = TomcatJSS.getInstance();
-        tomcatjss.loadConfig();
-        tomcatjss.init();
-
-        String subsystem = parent.parent.parent.getName();
-        String configFile = catalinaBase + File.separator + subsystem + File.separator +
-                "conf" + File.separator + CMS.CONFIG_FILE;
-
-        logger.info("Loading " + configFile);
-        ConfigStorage storage = new FileConfigStore(configFile);
-        EngineConfig cs = new EngineConfig(storage);
+        super.initializeTomcatJSS();
+        EngineConfig cs = super.getEngineConfig(parent.getParent().getParent().getParent().getName());
```

Comment:
I think there's too many `parent/getParent()` here. I'd suggest we restore the original `subsystem` variable and just call `getEngineConfig(subsystem)`. Also, it's not necessary to use `super.` on `initializeTomcatJSS()` and `getEngineConfig()`.

Reply:
ACK